tests/integration/ceph_base_file_test.go

```diff
@@ -64,6 +274,11 @@ func runFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.S
 	err = writeAndReadToFilesystem(helper, k8sh, s, namespace, filePodName, "test_file")
 	assert.NoError(s.T(), err)
 
+	if useCSI {
+		fileSystemCSISnapshotTest(helper, k8sh, s, storageClassName, namespace)
+		fileSystemCSICloneTest(helper, k8sh, s, storageClassName, namespace)
+
```

Comment:
nit: no blank line before closing bracket, also check other updates to this PR for the same